Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -305,6 +305,38 @@ mode would look as follows:
         --set global.k8sServiceHost=API_SERVER_IP \\
         --set global.k8sServicePort=API_SERVER_PORT
 
+NodePort XDP Acceleration
+*************************
+
+Cilium has built-in support for accelerating NodePort, ExternalIPs and LoadBalancer
+services for the case where the arriving request needs to be pushed back out of the
+node when the backend is located on a remote node. This acting as a hairpinned load
+balancer can be handled by Cilium at the XDP (eXpress Data Path) layer where BPF is
+operating directly in the networking driver instead of a higher layer.
```

Comment:
```suggestion
Cilium has built-in support for accelerating NodePort, ExternalIPs and LoadBalancer
services for the case where the arriving request needs to be pushed back out of the
node when the backend is located on a remote node. This ability to act as a hairpin load
balancer can be handled by Cilium at the XDP (eXpress Data Path) layer where BPF is
operating directly in the networking driver instead of a higher layer.
```